### PR TITLE
dockapps for windowmaker

### PR DESCRIPTION
**Pull request by bstrik**
added some dockapps for windowmanager windowmaker:
- clock
- systemtray

### CODE REVIEW

File: pkgs/applications/window-managers/windowmaker/dockapps/wmcalclock.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+stdenv.mkDerivation {
+  name = "wmCalClock-1.25";
+  src = fetchurl {
+     url = http://www.cs.mun.ca/~gstarkes/wmaker/dockapps/files/wmCalClock-1.25.tar.gz;
+     sha256 = "4b42b55bb7c1d7c58b5ee1f0058c683d3e4f3e3380d3a69c54a50b983c7c1b3f";
+  };
+
+  buildInputs = [ pkgconfig libX11 libXpm libXext ];
+
+  postUnpack = "sourceRoot=\${sourceRoot}/Src";
```
Comment (jagajaga) [inline pkgs/applications/window-managers/windowmaker/dockapps/wmcalclock.nix]: So why backslash here?

**Comment by bstrik** [comment]
The backslash is needed here otherwise I get an "undefined variable sourceRoot".
I took this from existing packages in the nixpkgs repository.
If there is a better way please let me know.

**Comment by jagajaga** [comment]
You've run it locally? It's working? :)

**Comment by bstrik** [comment]
Yes I did run it locally and it works.

**Comment by jagajaga** [comment]
cc @AndersonTorres

**Comment by lethalman** [comment]
Travis passed, thanks.

**Pull request merged**